nautobot/dcim/tests/test_models.py

```diff
@@ -1423,6 +1429,61 @@ def test_create_cable_with_missing_status_connected(self):
 
         self.assertTrue(Cable.objects.filter(id=cable.pk).exists())
 
+    def test_deleting_device_with_multiple_types_of_connected_interfaces_successful(self):
+        Cable.objects.all().delete()
```

Comment:
Why is this needed?

Reply:
This is more or less for me trying to start the test with a clean slate. It was necessary because I ran into duplicate cables when creating new cables.